state/votes/api.ts

```diff
@@ -8,6 +8,7 @@ const listVotes = async (): Promise<VotesResponse> => {
 
   checkResponse(response);
 
+  // eslint-disable-next-line total-functions/no-unsafe-type-assertion
   return response.json() as Promise<VotesResponse>;
```

Comment:
Looks like I'm going to have to pull in [io-ts](https://github.com/gcanti/io-ts)!